NSS-Likeable-Lemurs/BangazonSite
# Addpaymenttype2order

## Link to Ticket
Closes # [Link](https://github.com/NSS-Likeable-Lemurs/BangazonSite/issues/31)

## Description of Proposed Changes
- When an authenticated user is viewing an open order, they can click on the details, and click on "complete order." this will take them to a view so that they may choose a payment type.

## Steps to Test

```sh
git fetch --all
git checkout <addpaymenttype2order>
1. log in
2. click on "view cart"
3. click on "details"
4. click on "complete order"
5. select payment type
6. click on "finalize payment"
Go back to the index and see that the paymentTypeId will be shown in the list
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* PaymentTypeViewModel/checkout.cshtml/OrdersController/_LoginPartial
## Mentions @kellycook301 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
